modules/face/src/predict_collector.cpp

```diff
@@ -0,0 +1,60 @@
+/*
+* Copyright (c) 2011,2012. Philipp Wagner <bytefish[at]gmx[dot]de>.
+* Released to public domain under terms of the BSD Simplified license.
+*
+* Redistribution and use in source and binary forms, with or without
+* modification, are permitted provided that the following conditions are met:
+*   * Redistributions of source code must retain the above copyright
+*     notice, this list of conditions and the following disclaimer.
+*   * Redistributions in binary form must reproduce the above copyright
+*     notice, this list of conditions and the following disclaimer in the
+*     documentation and/or other materials provided with the distribution.
+*   * Neither the name of the organization nor the names of its contributors
+*     may be used to endorse or promote products derived from this software
+*     without specific prior written permission.
+*
+*   See <http://www.opensource.org/licenses/bsd-license>
+*/
```

Comment:
I think this license is wrong. Please replace it with text from [the main license](https://github.com/Itseez/opencv/blob/master/LICENSE).